.github/workflows/node.js.yml

```
@@ -0,0 +1,48 @@
+# Workflow to run unit tests from Github Actions.
+# Inspired from: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

```suggestion
name: Dashboard
```

This name will be more helpful in the list of jobs.